library/junos_install_config

```diff
@@ -302,6 +308,11 @@ def _load_via_netconf(module):
                 if args['confirm'] is not None:
                     opts['confirm'] = args['confirm']
 
+                if args['check_commit_wait']:
+				   check_commit_wait = int(args['check_commit_wait'])
```

Comment:
@jNicker I merged pull request #156 to fix this issue. Thanks for the submission!